lib/package-lock.json

```diff
@@ -480,7 +480,6 @@
             "requires": {
                 "anymatch": "1.3.2",
                 "async-each": "1.0.1",
-                "fsevents": "1.1.3",
```

Comment:
Out of curiosity @nickrflynn - are the three changes to package-lock.json, package.json, etc intentional?

Reply:
sometimes when you run `npm install` it changes this file, especially if the collaborator are in two different enviroments ie. mac and windows. npm tries its best to keep it consistent but fails alot